.github/workflows/build_reusable.yml

```diff
@@ -135,6 +135,10 @@ jobs:
       - run: fnm use --install-if-missing ${{ inputs.nodeVersion || env.NODE_LTS_VERSION }}
       - run: fnm default ${{ inputs.nodeVersion || env.NODE_LTS_VERSION }}
       - run: node -v
+      - name: Prepare corepack
+        if: ${{ contains('ubuntu-latest', fromJson(inputs.runs_on_labels)) }}
```

Comment:
This is missing the versioned labels (e.g. `ubuntu-24.04`) but we're not using those and it's not meant as a permanent fix anyway. Only until GH updates their runner images.